SpatialGDK/Source/SpatialGDK/Public/SpatialView/OutgoingComponentMessage.h

```diff
@@ -0,0 +1,96 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#pragma once
+#include "SpatialView/ComponentData.h"
+#include "SpatialView/ComponentUpdate.h"
+#include "SpatialView/EntityComponentId.h"
+
+namespace SpatialGDK
+{
+
+// Represents one of a component addition, update, or removal.
+// Interally schema data is stored using raw pointers. However the interface exclusively uses explicitly owning objects to denote ownership.
+// Not safe to use after being moved from.
```

Comment:
Let's put it in a default state nonetheless. At least it will allow us to assert when such an incorrect use is detected.

Reply:
As part of putting the NONE state in it now has a well defined post-move state.